po/fr.po

```diff
@@ -153,15 +155,12 @@ msgid ""
 "that if another Gnome Shell component clones the mouse (like the Zoom "
 "accessibility feature), the cursor will follow the screen brightness."
```

Comment:
The reason the translated string doesn't show is because the source string in the XML file has changed.  The current string looks like this:

```
#: schemas/org.gnome.shell.extensions.soft-brightness-plus.gschema.xml:53
msgid ""
"When enabled, the mouse cursor follows the brightness setting. When "
"disabled, the mouse cursor always remains at full brightness. Controlling "
"mouse cursor brightness can sometimes show the wrong cursor and introduce "
"cursor lag. You may want to disable it if you encounter cursor issues. Note "
"that when other Gnome Shell components clone the mouse (like the Zoom "
"accessibility feature), this feature may not work correctly, or there may "
"be multiple cursors shown on the screen."
msgstr ""
```

Would you mind updating the French translation to match?